test/functional/interface_http.py

```diff
@@ -15,6 +15,7 @@ def set_test_params(self):
         self.num_nodes = 3
 
     def setup_network(self):
+        self.extra_args =[["-rpccorsdomain=null"], [], []]
```

Comment:
Nit: Missing space after `=` :-)

Reply:
Weird, it was fixed in 20d3c18964dff877c2fbf7189819412289eaaab2 but still shows without the space.

Reply:
My bad. Fixed in 210b8ee1fbf79cff98bf76d53b4f35bc9cdd02d2